DEHPEcosimPro/Enumerators/MappingDirection.cs

```diff
@@ -0,0 +1,42 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MappingDirection.cs" company="RHEA System S.A.">
+//    Copyright (c) 2020-2020 RHEA System S.A.
+// 
+//    Author: Sam Gerené, Alex Vorobiev, Alexander van Delft, Nathanael Smiechowski.
+// 
+//    This file is part of DEHPEcosimPro
+// 
+//    The DEHPEcosimPro is free software; you can redistribute it and/or
+//    modify it under the terms of the GNU Lesser General Public
+//    License as published by the Free Software Foundation; either
+//    version 3 of the License, or (at your option) any later version.
+// 
+//    The DEHPEcosimPro is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//    Lesser General Public License for more details.
+// 
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with this program; if not, write to the Free Software Foundation,
+//    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace DEHPEcosimPro.Enumerators
+{
+    /// <summary>
+    /// Enumeration of the two direction a mapping can be done
+    /// </summary>
+    public enum MappingDirection
+    {
+        /// <summary>
+        /// Left means from EcosimPro to 10-25 source
```

Comment:
the names are not clear and need docs to be explanatory, i find the left-to-right idea a bit odd. Can you make the directionality more explicit please? And, is this not something that is more for Common? All DST adaptors will have a MappingDirection where on the one side we have 10-25 and the other we have a DST.